src/card/card-header.jsx

```diff
@@ -94,14 +94,15 @@ const CardHeader = React.createClass({
     let titleStyle = this.prepareStyles(styles.title, this.props.titleStyle);
     let subtitleStyle = this.prepareStyles(styles.subtitle, this.props.subtitleStyle);
 
-    let avatar = this.props.avatar;
+    let avatar = this.props.avatar || null;
```

Comment:
What if you don't set a default at all and just check on line 102 for 
` else if (typeof this.props.avatar === 'string') { `

Seems to express the intention of the if-statement better, imo.